src/main-process/atom-application.js

```diff
@@ -23,6 +23,20 @@ const ConfigSchema = require('../config-schema')
 
 const LocationSuffixRegExp = /(:\d+)(:\d+)?$/
 
+const getDefaultPath = () => {
+  const editor = atom.workspace.getActiveTextEditor()
+  if (!editor) {
```

Comment:
Seems like this can be shortened to `!editor || !editor.getPath()`

Reply:
Yep